packages/http-client-python/eng/scripts/setup/prepare.py

```diff
@@ -27,7 +27,7 @@ def main():
     venv_context = create_venv_with_package_manager(venv_path)
 
     try:
-        install_packages(["-r", f"{_ROOT_DIR}/generator/dev_requirements.txt"], venv_context)
```

Comment:
for relative dependency file, we had better add `cwd` to make sure the execute path is absolutely right.